MC/analysis_testing/o2dpg_analysis_test_workflow.py

```diff
@@ -428,6 +452,7 @@ def main():
     parser.add_argument("--period-name", dest="period_name", help="period name")
     parser.add_argument("--config", help="overwrite the default config JSON. Pass as </path/to/file>, will be automatically configured to json://")
     parser.add_argument("--only-analyses", dest="only_analyses", nargs="*", help="filter only on these analyses")
+    parser.add_argument("--autoset-converters", dest="autoset_converters", action="store_true", help="Compatibility mode to automatically set the converters for the analysis")
```

Comment:
Thanks @njacazio it looks very good to me! Just for completeness: I'd rename `autoset_converters` to something like `autoset_collision_converter` or something similar

Reply:
I think, this option makes sense as is. We might need to add other converters in the future. I think this is what was foreseen here?!

Reply:
No, these should be general, it could also work on e.g. mcParticles and so on

Reply:
Yes, thanks @benedikt-voelkel

Reply:
apologize, you are right, it makes sense as it is because the proper converter is switched-on if a table is not found